Dockerfile

```diff
@@ -9,14 +9,18 @@ MAINTAINER Enric Tejedor Saavedra <enric.tejedor.saavedra@cern.ch>
 RUN yum -y install yum-plugin-ovl # See https://github.com/CentOS/sig-cloud-instance-images/issues/15
 
 # Disable requiretty and secure path - required by systemuser.sh
-RUN yum -y install sudo
-RUN sed -i'' '/Defaults \+requiretty/d'  /etc/sudoers
-RUN sed -i'' '/Defaults \+secure_path/d' /etc/sudoers
+RUN yum -y install sudo && \
+    sed -i'' '/Defaults \+requiretty/d'  /etc/sudoers && \
+    sed -i'' '/Defaults \+secure_path/d' /etc/sudoers && \
+    yum clean all && \
+    rm -rf /var/cache/yum
 
 # Install ROOT prerequisites
 RUN yum -y install \
     libXpm \
-    libXft
+    libXft && \
+    yum clean all && \
+    rm -rf /var/cache/yum
```

Comment:
Does it make sense to do the cleaning only once? Doesn't it compensate, the cleaning at the end, for all the accumulated content in the previous layers?

Reply:
If you do not clean after each command, the layer will have the temp files. So cleaning only at the end does nothing to prevent intermediate layers from having those files, therefore making the final image bigger.